packages/browser/package.json

```diff
@@ -27,10 +27,7 @@
     "build-docs-preview-site": "npm run build-api-docs; cd docs/api; make html"
   },
   "devDependencies": {
-    "cross-fetch": "^3.1.5",
-    "webpack": "^5.11.0",
-    "webpack-cli": "^4.2.0",
-    "webpack-merge": "^5.7.2"
```

Comment:
@NSeydoux the duplication between here and the root `package.json` file was causing additional audit failures, despite your earlier fix. Removing these from here and upgrading the parent package's versions resolves the issue.